### PR TITLE
fix: addresses the delegate staked t functionality issue

### PR DESCRIPTION
**Pull request by evandrosaturnino**
## Description 

- [X] - [Remove walletconnect and add web3onboard to address issue with wallet connection in some devices](https://github.com/threshold-network/threshold-delegate/commit/333be51302418af09786e64ea3a4fdc36f0c08c9)
- [X] - [Update delegateVoting function to enable delegating staked t functionality](https://github.com/threshold-network/threshold-delegate/commit/18f9fd870f3594f23b2f431869f7555c3b17a490)

## Notice 

- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/threshold-network/threshold-delegate/pulls) for the same update/change? 

## Pull Request Type 

- [x] Fix (Non-breaking Change) 

## Issue (if applicable) 

Closes issue #2 
Closes issue #3 

## Testing Please outline all testing steps 

Testing locally: 
1. Clone the repo locally: `git clone https://github.com/threshold-network/threshold-delegate.git` 
2. Install the dependecies with yarn: `yarn install`
3. Add a REACT_APP_WALLETCONNECT_ID variable in .env file with your wallet connect ID
4. Build the app: `yarn build` 
5. Run the app: `yarn start`

### CODE REVIEW

File: src/components/Modals/DelegateModal/DelegateModal.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {
 	useToast,
 } from '@chakra-ui/react';
 import { errorToast, infoToast, successToast } from '../../../utils/toastify';
+import { useConnectWallet } from '@web3-onboard/react';
```
Comment (manumonti) [inline src/components/Modals/DelegateModal/DelegateModal.jsx]: This line can be deleted.

File: src/components/Navigation/Footer/Footer.jsx [hunk 2]
```diff
@@ -8,8 +8,6 @@ import { FaInstagram, FaTwitter, FaYoutube } from "react-icons/fa";
  * @param {string} children - The children (Icon).
  * @param {string} label - The label (VisuallyHidden)
  * @param {string} href - The href - Destination URL.
```
Comment (manumonti) [inline src/components/Navigation/Footer/Footer.jsx:10]: Maybe we can delete the rest of mentions to author in this file, and also in `Delegates.jsx` and `ColorModeSwitcher.jsx`. Not a blocker.

**Review by manumonti — APPROVED** [review]
LGTM 🙌! I left some comments, but any of them is a blocker.

Also, I would suggest to update the README with instructions about how to start the dev page now that we are using `yarn`. But we can do this afterwards.

File: src/components/Navigation/Footer/Footer.jsx (hunk 2 above)
Comment (evandrosaturnino) [reply src/components/Navigation/Footer/Footer.jsx:10]: Addressed in https://github.com/threshold-network/threshold-delegate/pull/4/commits/f616943f5b00a9f401a57fe0df8acdf2896144d6

File: src/components/Modals/DelegateModal/DelegateModal.jsx (hunk 1 above)
Comment (evandrosaturnino) [reply src/components/Modals/DelegateModal/DelegateModal.jsx]: Addressed in https://github.com/threshold-network/threshold-delegate/pull/4/commits/4c85d80ebcfa05bcf04fed1e2ee68edd9d2ed0df

**Pull request merged**